webapp/templates/team/partials/clarification_list.html.twig

```diff
@@ -44,7 +44,7 @@
                 <td>
                     <a data-ajax-modal data-ajax-modal-after="markSeen" href="{{ link }}">
                         {%- if clarification.problem -%}
-                            problem {{ clarification.problem.contestProblems.first.shortname -}}
+                            problem {{ clarification.problem.contestProblems.first | problemBadge -}}
```

Comment:
We should dot his on the jury clar list as well then

Reply:
True, I'll look into it after lunch